app/components/SvgTemplate.tsx

```diff
@@ -47,6 +47,7 @@ export interface StoneSettings {
   blueExp: number;
   blueOff: number;
   rotation: number;
+  seed: number;
```

Comment:
It'd be great to use an account address for the seed, as that would let us show how the seed affects the stone in the minting app when a wallet is connected.